input/europe/documentation

```diff
@@ -0,0 +1,9 @@
+eu_very_crude
```

Comment:
I guess I was trying to keep the Europe folder separate from the old files.. But that makes total sense! I will learn how to do in the write_reactors.py file!